build.sbt

```diff
@@ -68,6 +68,7 @@ lazy val server = (projectMatrix in file("server"))
       libraryDependencies ++= Dependencies.serverDependencies,
       Compile / packageBin / publishArtifact := false,
       (Compile / compile) := ((Compile / compile) dependsOn printSparkScalaVersion).value,
+      crossScalaVersions := Seq(Versions.serverScalaVersion),
```

Comment:
What is the purpose of this setting?

Reply:
hmm, okay I think it was a remnant from the past, will delete it